inspector/src/components/actionTabs/tabs/propertyGrids/meshes/skeletonPropertyGridComponent.tsx

```diff
@@ -166,38 +166,38 @@ export class SkeletonPropertyGridComponent extends React.Component<ISkeletonProp
+                    <FloatLineComponent label="midStep" target={this._skeletonViewerDisplayOptions} propertyName='midStep' onPropertyChangedObservable={this.props.onPropertyChangedObservable} onChange={(value) => { this.changeDisplayOptions('midStep', value) }} />
+                    <FloatLineComponent label="midStepFactor" target={this._skeletonViewerDisplayOptions} propertyName='midStepFactor' onPropertyChangedObservable={this.props.onPropertyChangedObservable} onChange={(value) => { this.changeDisplayOptions('midStepFactor', value) }} />
+                </LineContainerComponent>)
+        } else {
+            displayOptions = (null)
         }
 
         return (
             <div className="pane">
                 <CustomPropertyGridComponent globalState={this.props.globalState} target={skeleton}
                     lockObject={this.props.lockObject}
-                    onPropertyChangedObservable={this.props.onPropertyChangedObservable} />                    
+                    onPropertyChangedObservable={this.props.onPropertyChangedObservable} />
                 <LineContainerComponent title="GENERAL" selection={this.props.globalState}>
                     <TextLineComponent label="ID" value={skeleton.id} />
                     <TextLineComponent label="Bone count" value={skeleton.bones.length.toString()} />
                     {
                         skeleton.overrideMesh &&
-                        <TextLineComponent label="Override mesh" value={skeleton.overrideMesh.name} onLink={() => this.onOverrideMeshLink()}/>
-                    }                        
+                        <TextLineComponent label="Override mesh" value={skeleton.overrideMesh.name} onLink={() => this.onOverrideMeshLink()} />
+                    }
                     <CheckBoxLineComponent label="Use texture to store matrices" target={skeleton} propertyName="useTextureToStoreBoneMatrices" onPropertyChangedObservable={this.props.onPropertyChangedObservable} />
-                    
-                    <LineContainerComponent title="DEBUG" selection={this.props.globalState}>                        
-                        <CheckBoxLineComponent label="Enabled" isSelected={() => this._skeletonViewersEnabled} onSelect={() => this.switchSkeletonViewers()} />
-                        <OptionsLineComponent label="displayMode" options={debugModeOptions} target={this._skeletonViewerDisplayOptions} propertyName="displayMode" onPropertyChangedObservable={this.props.onPropertyChangedObservable} onSelect={() => this.changeDisplayMode()} />
-                        {displayOptions}                   
-                    </LineContainerComponent>                    
+                    <ButtonLineComponent label="Return to rest" onClick={() => skeleton.returnToRest()} />
+                </LineContainerComponent>
```

Comment:
looks like it is now 2 lines instead of a line in a line

As I have no idea which one it should be, just raising here in case it was not intended even if I am pretty sure the new way is the right one  :-)

Reply:
Yeah, I noticed this, thought it was weird, and changed it. The display result is barely noticeable.